views/home/main.js

```diff
@@ -1,8 +1,32 @@
+let toggles_div = document.getElementById("toggles-div");
 let qrcode_div = document.getElementById("qrcode-div");
 let qrcode_submit = document.getElementById("qr_submit");
-let url = document.getElementById("url");
 let size = document.getElementById("size");
 
+// These social medias will be pulled from the DB
+let social_medias = ["Instagram", "Facebook", "LinkedIn", "Email", "Number"];
+// URL will be changed to the page that displays all of the info of the user
+let url = "http://www.google.com";
+
+for (let i = 0; i < social_medias.length; i++) {
+  let toggle_div = document.createElement("div");
+  toggle_div.classList.add("form-check", "form-switch", "form-switch-xl");
+  let toggle = document.createElement("input");
+  toggle.classList.add("form-check-input", "col-4");
+  toggle.type = "checkbox";
+  toggle.role = "switch";
+  toggle.id = social_medias[i];
```

Comment:
All good stuff here. One addition that we will do once we are ready to connect the front end with the API is to make sure that the id that comes in with each label/value pair for an account from the database is added somewhere in these toggle switches. 

You'll see that the API will eventually expect an array of `user_account` ids when generating the QR code, so they need to be stored with each input toggle switch. It may be best to store them in the input's `id` attribute, but for now, this is fine for the dummy data. Mainly noting this here so we know what to replace when it's time.